test/runnable/cppa.d

```diff
@@ -727,6 +727,19 @@ void test16()
     static assert(0);
 }
 
+/****************************************/
+/+ FIXME: Requires C++11 compiler.
```

Comment:
I think the time has come to segment tests by C++ version.
You can create a separate file, let's say `cppa_11.d`, and use:
```
 // EXTRA_CPP_SOURCES: cppa_11.cpp
// CXXFLAGS: -std=c++11
```

To test C++11-specific behaviors

Reply:
Nice idea though it can probably be done as another PR.

Reply:
I don't think we should be rushing to merge PRs without any tests in them.
And it's not a large change to do either, moving a bit of code to a new file, so I really don't understand the hurry.